torch2jax/__init__.py

```diff
@@ -1,13 +1,25 @@
 import copy
 import functools
 import math
-from typing import Optional
+from typing import (Any, Optional, Sequence, Union, Tuple)
+
+import numpy as np
```

Comment:
Is there any way we can avoid introducing a dependency on numpy? Why not use `jax.numpy` instead?